HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,795 @@
+			"description": "Sets the MIME type for the 'image' resource."
+		},
+		"format": {
+			"type": "string",
+			"description": "Name of the format or schema used by the NFT."
+		},
+		"properties": {
+			"type": "object",
+			"properties": {
+				"external_url": {
+					"type": "string",
+					"format": "uri",
+					"description": "A URI pointing to an informational page about the NFT."
+				}
+			},
+			"additionalProperties": true
+		},
+		"files": {
+			"type": "array",
+			"contains": {
+				"type": "object"
+			},
+			"minContains": 1,
+			"items": {
+				"type": "object",
+				"properties": {
+					"uri": {
+						"type": "string",
+						"format": "uri",
+						"description": "A URI pointing to a resource."
+					},
+					"sha256_checksum": {
+						"type": "string",
+						"description": "Cryptographic hash of the representation of the 'uri' resource."
+					},
+					"type": {
+						"type": "string",
+						"description": "Sets the MIME type for the 'image' resource."
+					},
+					"is_default_file": {
+						"type": "boolean",
+						"description": "Indicates if this file object is the main file representing the NFT."
+					},
+					"metadata": {
+						"type": "object",
+						"description": "Represents a nested metadata object for the file."
+					},
+					"metadata_uri": {
+						"type": "string",
+						"format": "uri",
+						"description": "A URI pointing to a metadata resource."
+					},
+				},
+				"required": [
+					"uri",
+					"type"
+				],
+				"additionalProperties": false
+			}
+		},
+		"attributes": {
+			"type": "array",
+			"items": {
+				"type": "object",
+				"properties": {
+					"trait_type": {
+						"type": "string",
+						"description": "Name of trait."
+					},
+					"display_type": {
+						"type": "string",
+						"description": "Sets the representation of the value of the trait."
+					},
+					"value": {
+						"type": ["string", "integer", "number", "boolean"],
+						"description": "Value for trait."
+					},
+					"max_value": {
+						"type": "string",
+						"description": "Maximum value for trait."
+					}
+				},
+				"required": [
+					"trait_type",
+					"value"
+				],
+				"additionalProperties": false
+			}
+		},
+		"localization": {
+			"type": "array",
+			"items": {
+				"type": "object",
+				"properties": {
+					"locale": {
+						"type": "string",
+						"description": "Sets the two-letter language code for the localization resource."
+					},
+					"is_default_locale": {
+						"type": "boolean",
+						"description": "Indicates if the set locale is the default locale for this metadata file."
+					},
+					"metadata_uri": {
+						"type": "string",
+						"format": "uri",
+						"description": "A URI pointing to a metadata resource."
+					}
+				},
+				"required": [
+					"locale"
+				],
+				"additionalProperties": false
+			}
+		}
+	},
+	"required": [
+		"name",
+		"creator",
+		"image",
+		"type",
+		"format",
+		"files",
+		"properties",
+		"attributes"
```

Comment:
I think these would be the minimum required properties for a very simple NFT. I think we're OK adding image and type as required now if we're pushing for it to be the preview image. I can't remember the react exact reason we made image optional in HIP 412, feel free to chime in if anyone sees a reason why this shouldn't be required.

```suggestion
		"name",
		"image",
		"type"
```

Reply:
As this will become a reference implementation under HIP412, this should be specific. But you are right about setting `properties` and `attributes` to be optional. The rest should be required.

Reply:
“image” was originally listed as ‘required’, however HashAxis noted that in the case of file NFTs, creating both image and file would require two files to pin instead of one. At the scale of millions, this doubling of the number of pinned images is significantly impactful to the cost of hosting the files. For files that can have their thumbnails generated programmatically, an image is not required. Because of this, the “image” field was changed to Conditionally Optional.